actions/views/profile_popover.js

```diff
@@ -8,21 +8,21 @@ import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
 
 import {getSelectedPost} from 'selectors/rhs';
 
-export function getMembershipForCurrentEntities(userId) {
+export function getMembershipForCurrentEntities(userId, channelId) {
     return async (dispatch, getState) => {
         const state = getState();
         const currentTeamId = getCurrentTeamId(state);
 
         const selectedPost = getSelectedPost(state);
         const currentChannel = getCurrentChannel(state) || {};
 
-        let channelId;
+        let defaultChannelId;
         if (selectedPost.exists === false) {
-            channelId = currentChannel.id;
+            defaultChannelId = currentChannel.id;
         } else {
-            channelId = selectedPost.channel_id;
+            defaultChannelId = selectedPost.channel_id;
         }
```

Comment:
minor nitpick, mostly to avoid a negative condition and to have a `const` instead of a `let` if there is no need to change it.
```js
const defaultChannelId = selectedPost.exists ? selectedPost.channel_id : currentChannel.id;
```